packages/cli/go.mod

```diff
@@ -30,6 +30,7 @@ require (
 	github.com/rsc/wes_client v0.0.0-00010101000000-000000000000
 	github.com/spf13/afero v1.6.0
 	github.com/spf13/cobra v1.2.1
+	github.com/spf13/pflag v1.0.5
```

Comment:
Why do we need this dependency? We already have cobra for cli args parsing.

Reply:
This is an implicit dependency but still required in the go.mod file otherwise go build fails.